### DISCUSSION TITLE
TypeError: C:\Users\MY HP\Desktop\projectt updating\my_project\client\polyfills.js: this.setDynamic is not a function

### DISCUSSION THREAD

**Discussion by Er-Ramanand-Dubey** [vercel/next.js#16377] [Help]
can anyone help me please,
my polyfillsa.js file looks like this.

`import includes from 'core-js/library/fn/string/virtual/includes'

import repeat from 'core-js/library/fn/string/virtual/repeat'

import assign from 'core-js/library/fn/object/assign'

import from from 'core-js/library/fn/array/from'

console.log('Load your polyfills')

String.prototype.includes = includes

String.prototype.repeat = repeat

Object.assign = assign

Array.from = from

// import 'core-js/features/array/includes'
// import 'core-js/features/string/repeat'
// import 'core-js/features/array/from'
// import assign from 'object-assign'
// Object.assign = assign

`

**Comment by jamesmosier** [comment]
Could you provide a GitHub repository link with the code that is causing you problems?

If you update to the latest version of Next (or any version past 9.3) these polyfills are all included for you, so you could remove this polyfills file.
